.github/workflows/ci.yml

```diff
@@ -253,3 +280,244 @@ jobs:
           git push -u origin $env:GH_DOC_BRANCH --force
         env:
           GH_DOC_BRANCH: gh-pages
+
+  build_test_package_others:
+    name: Build Test and Package for release
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2019]
+        python-version: ['3.7', '3.9']
+
+    env:
+      ANSYS_VERSION: 221
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2.1.4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - id: install-dpf
+        uses: pyansys/pydpf-actions/install-dpf-server@v1
+        with:
+          dpf-standalone-TOKEN: ${{secrets.DPF_PIPELINE}}
+          ANSYS_VERSION : ${{env.ANSYS_VERSION}}
+
+      - name: Set AWP_ROOT$env:ANSYS_VERSION
```

Comment:
@PProfizi this should already be set in the pydpf-cactions by the lines:
```
   - run: echo "AWP_ROOT${{inputs.ANSYS_VERSION}}=${{env.SERVER}}"  | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
      shell: pwsh
```

Reply:
@cbellot000 Ok, noted, I'll remove it and check.